### PR TITLE
Release 2.5.4-beta2

### PR DESCRIPTION
**Pull request by niik**
## Description

Looking for the PR for the upcoming v2.5.4-beta2 beta release? Well you've just found it, congratulations! :tada:

## Release checklist

Release checklist:

- [x] ~~Check to see if there are any errors in Sentry that have only occurred since the last production release~~
- [x] Verify that all feature flags are flipped appropriately
- [x] If there are any new metrics, ensure that central and desktop.github.com have been updated

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,20 @@
 {
   "releases": {
+    "2.5.4-beta2": [
+      "[Added] Suggest to stash changes when trying to do an operation that requires a clean working directory - #10053",
+      "[Fixed] Pull request list is now keyboard accessible - #10334",
+      "[Fixed] Correct branch name in commit button when on an unborn branch - #10318",
+      "[Fixed] Fix \"Show in Explorer\" in Windows when there are other files with the same name as the project folder. - #10195",
```
Comment (rafeca) [inline changelog.json:7]:
```suggestion
      "[Fixed] Fix \"Show in Explorer\" in Windows when there are other files with the same name as the project folder - #10195",
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,20 @@
 {
   "releases": {
+    "2.5.4-beta2": [
+      "[Added] Suggest to stash changes when trying to do an operation that requires a clean working directory - #10053",
```
[the remark this answers is not in the document]
Comment (niik) [reply changelog.json:4]: Thanks for the suggestion @say25 (see what I did there?)

When reading the changelog in this diff I agree that your suggestion makes more sense but when looking at the release notes in the app or on desktop.github.com the `Added`/`Improved`/`Fixed` doesn't appear to be part of the sentence structure like it does here.

image

It's a totally subjective call but I'm gonna stick with the original wording here.

[the remark this answers is not in the document]
Comment (rafeca) [reply changelog.json:4]: FWIW, we have [a styleguide](https://github.com/desktop/desktop/blob/development/docs/process/writing-release-notes.md#verbiage) about writing release notes, which specifically talks about not using the tag prefix as part of the sentence.

**Comment by niik** [comment]
This shipped today!

**Pull request merged**